simphony/cuds/abc_modeling_engine.py

```diff
@@ -37,16 +37,9 @@ def add_lattice(self, lattice):
 
         Parameters
         ----------
-        lattice: ABCLattice
+        lattice : ABCLattice
             lattice to be added.
 
-        Returns
```

Comment:
If I am correct, adding a lattice into a lattice container should not return anything.